doc/man1/ciphers.pod

```diff
@@ -167,7 +167,8 @@ The following is a list of all permitted cipher strings and their meanings.
 The default cipher list.
 This is determined at compile time and is normally
 B<ALL:!COMPLEMENTOFDEFAULT:!eNULL>.
-When used, this must be the first cipherstring specified.
+When used, this must be the first cipherstring specified and it cannot be
+prefixed by any of the special characters described above.
 
```

Comment:
Not just "cannot be prefixed": even `DEFAULT+ECDH` will not behave as other cipher string would (ie. select what is in DEFAULT *and* uses ECDH).

Reply:
That is not what + means.  Try these:
```
openssl ciphers -v DEFAULT | tail
openssl ciphers -v DEFAULT+RSA | tail
```
It does not do an AND.  It does a "move to end of the list."

Reply:
You can do RSA+AES, where + does mean and. DEFAULT+AES does not work because of the bug. On the other hand ALL+RSA:!COMPLEMENTOFDEFAULT:!eNULL works.